src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carCostPerDay = 40;
+  const mediumRentTermDiscount = 20;
+  const longRentTermDiscount = 50;
```

Comment:
The variable names are descriptive, but the values should be defined as constants outside of the function to avoid hardcoding. This will make it easier to change the values in one place if needed.